generator/.DevConfigs/A78BAE5D-9864-4E0F-853F-BBA0ACD3F60D.json

```diff
@@ -0,0 +1,12 @@
+{
+    "core": {
+        "updateMinimum": true,
+        "type": "Patch",
+        "changeLogMessages": [
+            "Add the ConnectTimeout property on the service client config for the .NET 8 target of the SDK."
+        ],
+		"backwardIncompatibilitiesToIgnore": [
```

Comment:
why is this a breaking change for this pr but not in v3?

Reply:
It is in V3 as well but we don't run the assembly comparer in V3 build system anymore